src/pages/home/_index.tsx

```diff
@@ -32,10 +32,16 @@ export default function Home() {
             <div className={clsx('col col--12', styles.heroHeadline)}>
               <h1>{siteConfig.tagline}</h1>
               <Link
-                className="button button--electron button--dark button--lg"
+                className="button button--electron button--dark margin-right--sm"
                 to="/docs/latest/"
               >
-                Docs
+                Get Started →
+              </Link>
+              <Link
+                className="button button--electron button--outline button--dark margin-right--sm"
+                to="/docs/latest/why-electron"
+              >
+                Why Electron?
```

Comment:
Can we make those buttons larger?

The general rule of thumb for CTAs is that they should be 2x as big as you'd naturally make them.

Reply:
https://github.com/electron/website/pull/816/commits/56b74d37e6fb06f4c32b52e3908334df45a5bbfd